addons/sourcemod/scripting/scp_sf/items.sp

```diff
@@ -1746,25 +1750,16 @@ public Action Items_FragTimer(Handle timer, int ref)
 			SetVariantString("classname taunt_soldier"); 
 			AcceptEntityInput(explosion, "AddOutput");
 			
-			int particle = AttachParticle(explosion, "asplode_hoodoo", false, 5.0);
-			if (particle)
-				EmitGameSoundToAll("Weapon_Airstrike.Explosion", particle, entity, _, _, pos);
-				
+			AttachParticle(explosion, "asplode_hoodoo", false, 5.0);
+			EmitGameSoundToAll("Weapon_Airstrike.Explosion", entity);
+			
 			AcceptEntityInput(explosion, "Explode");
 			CreateTimer(0.1, Timer_RemoveEntity, EntIndexToEntRef(explosion), TIMER_FLAG_NO_MAPCHANGE);
 		}
-		
+
 		// find any doors nearby and try destroy or force them open
-		//CopyVector(pos, FragPos); // temporary for trace
-		
-		// FIXME: switch to this method when SM 1.11 is stablew
-		//TR_EnumerateEntitiesSphere(pos, 350.0, PARTITION_SOLID_EDICTS, Items_FragTrace);
-		
-		int entitytrace = -1;
-		while ((entitytrace = SDKCall_FindEntityInSphere(entitytrace, pos, 350.0)) != -1)
-		{
-			Items_FragTrace(entitytrace);
-		}
+		//CopyVector(pos, FragPos); // temporary for trace, used for pushing items away
```

Comment:
This comment can be removed.

Reply:
I left that commented line there because it goes along with the rest of the commented code here:

https://github.com/redsunservers/SCP-Secret-Fortress/blob/41cfa13671bbfcc5e3a27de2a612cf2d103e3d63/addons/sourcemod/scripting/scp_sf/items.sp#L1689-L1720

should all of it be removed?

Reply:
I think this can go yes, it's highly unlikely we're going to require the vphysics extension. Maybe if you want to implement it as an optional dependency, this might be useful